index.html

```diff
@@ -151,8 +180,70 @@ <h2 class ="speaker-title">MORE</h2>
                 </div>
             </button>
         </section>
-    </section>
 
-    
+        <section class="speakers-desktop">
+
+            <h3 class="speakers-desktop-title">Feature Speakers</h3>
+            <hr class="about-desktop-hr">
+
+            <div class="speakers-section-desktop">
+
+                <div class="box-speaker speaker-desktop-box">
+                    <div class="speaker-desktop-main">
+                    </div>
+                </div>
+
+                <div class="box-speaker speaker-desktop-box">
+                    <div class="speaker-desktop-main"></div>
+                </div>
+
+                <div class="box-speaker speaker-desktop-box">
+                    <div class="speaker-desktop-main"></div>
+                </div>
+
+                <div class="box-speaker speaker-desktop-box">
+                    <div class="speaker-desktop-main"></div>
+                </div>
+
+                <div class="box-speaker speaker-desktop-box">
+                    <div class="speaker-desktop-main"></div>
+                </div>
+
+                <div class="box-speaker speaker-desktop-box">
+                    <div class="speaker-desktop-main"></div>
+                    <br><br>
+                </div>
+
+                
+            
+            </div>
+
+        </section>
+
+        <section class="partners-section">
+            <h3 class="partners-desktop-title">Partners</h3>
+            <hr class="about-desktop-hr">
+            <ul class="partners-ul">
+                <img class="partners-img" src="img/ethereum.svg" alt="">
+                <img class="partners-img" src="img/synthetix.svg" alt="">
+                <img class="partners-img" src="img/storj.svg" alt="">
+                <img class="partners-solana-img" src="img/solana.svg" alt="">
+                <img class="partners-img" src="img/filecoin.svg" alt="">
+            </ul>
+
+        </section>
+        
+        <footer>
+            <div class="footer-content">
+                <img class="footer-logo" src="img/smartbuild.jpg" alt="">
+                <div class="footer-text">
+                    <p class="footer-general-info">2023 SmartBuild Conference. All Rights Reserved.</p>
+                    <p class="footer-general-content">SmartBuild is a conference targeting all over the world for introducing the recent updates in Web3 environment.
+                        The conference is also aiming to enable people get in touch with business leaders and create a network to discuss the newest changes.
+                        People can also attend to the lectures and join to the hackhathon from all over the world.
+                    </p>
+                </div>
+            </div>
+        </footer>
```

Comment:
- [ ] I think that the footer is a bit big, kindly reduce its height it would look better and it will match the design provided.